ming_sim/db.py

```
@@ -11190,11 +11205,10 @@ def read_dossier_fork_state(self, dossier_id: int) -> Dict[str, object]:
         execution_outcome = (
             str(target.get("execution_outcome") or "").strip() if target else ""
         )
-        moves = self.list_economy_moves_for_dossier(int(dossier_id))
-        actual_effect_count = len(moves) + len(
-            self.list_fiscal_effects_for_dossier(int(dossier_id))
-        )
-        beyond = any(bool(row.get("beyond_intent")) for row in moves)
+        # #1260：durable_effects / beyond 单源助手（economy+fiscal），禁再只扫 economy。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)